<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgSetting</class>
 <widget class="QDialog" name="DlgSetting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>334</width>
    <height>279</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Phone Login Settings</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="lblExtn">
         <property name="text">
          <string>Extension:</string>
         </property>
         <property name="buddy">
          <cstring>leExtn</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="leExtn"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="lblName">
         <property name="text">
          <string>Name:</string>
         </property>
         <property name="buddy">
          <cstring>leName</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="leName"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="lblPwd">
         <property name="text">
          <string>Auth:</string>
         </property>
         <property name="buddy">
          <cstring>lePwd</cstring>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="lePwd"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="lblServer">
         <property name="text">
          <string>Server Name:</string>
         </property>
         <property name="buddy">
          <cstring>leServer</cstring>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="leServer"/>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="lblURI">
         <property name="text">
          <string>URI:</string>
         </property>
         <property name="buddy">
          <cstring>leURI</cstring>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLineEdit" name="leURI"/>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Scheme:</string>
         </property>
         <property name="buddy">
          <cstring>leScheme</cstring>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QLineEdit" name="leScheme"/>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Proxy:</string>
         </property>
         <property name="buddy">
          <cstring>leProxy</cstring>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QLineEdit" name="leProxy"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="pbCreate">
         <property name="text">
          <string>Save</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pbCancel">
         <property name="text">
          <string>Cancel</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pbCancel</sender>
   <signal>clicked()</signal>
   <receiver>DlgSetting</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>201</x>
     <y>257</y>
    </hint>
    <hint type="destinationlabel">
     <x>305</x>
     <y>177</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
